client/lib/abtest/active-tests.js

```diff
@@ -157,5 +157,15 @@ module.exports = {
 		},
 		defaultVariation: 'disabled',
 		allowExistingUsers: true
+	},
+
+	readerSearchOnFollowing: {
+		datestamp: '20170206',
+		variations: {
+			show: 50,
+			hide: 50
+		},
+		defaultVariation: 'hide',
```

Comment:
The `defaultVariation` is used when a user is not eligible to participate in the A/B test.